card/VERSION

```diff
@@ -1 +1 @@
-1.14.8s
+1.15.pre
```

Comment:
Sounds good, I'll move it into the base directory.  One question I have is what to we call the repo base dir for development purposes?  That's the directory that will have this file and the one you'll need to supply to configure the gem path pointers for development mode.  It's one level up from the three gem dirs.